### ISSUE TITLE
Refactor: landing page

### ISSUE THREAD

**Issue by preschian** [kodadot/nft-gallery#3714]
Refactor components to use vue composition API. To make it easier for review and test, I will make multiple PR based on this list:

trying to refactor from the deepest/low-level component, in summary, carousel components are something like this:

```
carousel
├── card
│   └── media
│   └── identity
│       └── identity-popover
```

- [ ] [not started] curated list
- [ ] [not started] latest sales, popular collections, and newest list (move component from /rmrk/)
- [ ] [ongoing] identity - https://github.com/kodadot/nft-gallery/issues/3747
- [ ] [code review] media - https://github.com/kodadot/nft-gallery/pull/3718
- [ ] tbd...

to avoid losing so much context when reviewing the code, create a separate PR for refactoring to vue compose API. context: https://github.com/kodadot/nft-gallery/pull/3718#pullrequestreview-1072558888
- [ ] [start after #3747] migrate Identity to vue compose API - https://github.com/kodadot/nft-gallery/issues/3749
- [ ] [not started] carousel component

**Comment by kodabot** [comment]
ASSIGNMENT EXPIRED - @preschian has been unassigned.

**Comment by petersopko** [comment]
> ASSIGNMENT EXPIRED - @preschian has been unassigned.

@preschian you need to link the issues in PR if you want to keep the assignment going past the allocated time :)

**Comment by petersopko** [comment]
@preschian or should the PR solve this one only partially? perhaps we can extract the part you've done to standalone issue if that's the case?

**Comment by preschian** [comment]
> @preschian or should the PR solve this one only partially? perhaps we can extract the part you've done to standalone issue if that's the case?

oohh I think this is better. Yaps, this is only partial, next parts I will open a new standalone issue and link to this issue 👍

**Comment by yangwao** [comment]
lol why it closed even not linked as `pr closes` 🤔
thanks for reopening 🙌

**Comment by roiLeo** [comment]
> lol why it closed even not linked as `pr closes`

<img width="329" alt="Capture d’écran 2022-08-18 à 11 35 14 AM" src="https://user-images.githubusercontent.com/9987732/185363110-b24f8e99-f3e1-4e61-8207-ef91b36b387f.png">

linked issue in pr will close
